### ISSUE TITLE
Unable to access incus daemon from unprivileged user account

### ISSUE THREAD

**Issue by LOTRg** [lxc/incus#1010]
# Required information

 * Distribution: Fedora
 * Distribution version:  40
 * The output of "incus info" or if that fails:
   * Kernel version: 6.9.8-200.fc40.x86_64
   * LXC version:
   * Incus version: 6.3
   * Storage backend in use: btrfs

# Issue description

Listing container or exec into containers as root work without issues, however, with unprivileged user, it doesn't work.
Unprivileged user is part of the incus-admin group.
the error reads :

> "Error: The incus daemon doesn't appear to be started (socket path: /var/lib/incus/unix.socket)"

however the actual socket is in /run/incus/unix.socket, trying to list the permissions on the socket file as root shows 

> srw-rw----. 1 root incus-admin 0 Jul 17 13:32 /run/incus/unix.socket

as unpriviliged user, it says permission denied

>  ls: cannot access '/run/incus/unix.socket': Permission denied

 which seems to revert back to the old path /var/lib/incus/unix.socket as I understand from this 
https://github.com/lxc/incus/blob/main/client/connection.go#L183
Why the socket even though it has the correct permissions and the ownership is not accessible by the unprivileged account ?

This used to work up to version 6.2 but with the upgrade to version 6.3, it's now failing.

**Comment by LOTRg** [comment]
This can be closed .
The resolution for those who might be having this problem, 
the group incus-admin owning the path /run/incus needs to have full access permissions.
from 

> drwx------.  9 root    incus-admin  240 Jul 17 13:40 incus

to 
`chmod 770 /run/incus`
not sure why this happened after the upgrade but it's now resolved, hopefully it won't be overwritten on reboot or on next upgrade.

**Comment by LOTRg** [comment]
After a reboot or even a restart really of the incus service, the permissions are reverted.
the socket is owned by incus-admin group but the parent directory doesn't allow incus-admin group members to traverse the directory which prevents these accounts from finding the socket.

**Comment by LOTRg** [comment]
Seems this commit https://github.com/lxc/incus/commit/d69c680903eff251e987d90af729992d2e141f9c fixed the issue.

**Comment by LOTRg** [comment]
duplicate of https://github.com/lxc/incus/issues/1003

**Comment by billbalm** [comment]
Running Incus 6.2 on Fedora 41. Since the fix is for 6.8, one workaround until then is:

Chmod in ExecStartPost in /usr/lib/systemd/system/incus.service

```
ExecStart=/usr/libexec/incus/incusd --group incus-admin
ExecStartPost=/usr/libexec/incus/incusd waitready --timeout=600
ExecStartPost=chmod 711 /run/incus
```
